rust/src/bin/test-drive.rs

```diff
@@ -0,0 +1,56 @@
+use xain_fl::client::{Client, ClientError};
```

Comment:
This file become a test or an example in the future.

Reply:
added another sentence to the docstring to clarify purpose